scripts/cem-plugin-react-wrapper.js

```diff
@@ -0,0 +1,277 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import * as React from 'react';
+import { createComponent } from '@lit-labs/react';
+${componentImports.reduce((pre, cur) => pre + cur + '\n', '')}
+${fileImports.reduce((pre, cur) => pre + cur + '\n', '')}
+
+${reactComponents.reduce(
+    (pre, component) =>
+        pre +
+        `export const ${component.name} = createComponent({
+        displayName: '${component.name}',
+        elementClass: ${component.swcComponentName},
+        react: React,
+        tagName: '${component.elementName}',
+        events: {
+            ${component.events.reduce(
+                (pre, event) =>
+                    pre +
+                    `${event.name.replace(/-./g, (m) =>
+                        m[1].toUpperCase()
+                    )}: '${event.name}', `,
+                ''
+            )}
+        }
+    });`,
+    ''
+)}
+`;
+
+            const componentShortName = pkgName.replace(
+                '@spectrum-web-components/',
+                ''
+            );
+
+            const packageJson = `{
+    "name": "@swc-react/${componentShortName}",
+    "version": "${pkgVersion}",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "React wrapper of the ${pkgName} component",
+    "license": "Apache-2.0",
+    "author": "",
+    "main": "index.js",
+    "module": "index.js",
+    "type": "module",
+    "keywords": [
+        "React",
+        "Spectrum Web Components"
+    ],
+    "peerDependencies": {
+        "react": "^17.0.0 || ^18.0.0"
+    },
+    "dependencies": {
+        "@lit-labs/react": "1.1.0",
+        "${pkgName}": "${pkgVersion}"
+    }
+}
+`;
+
+            const tsconfigJson = `{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        "composite": true,
+        "rootDir": "./"
+    },
+    "include": ["*.ts"]
+}
+`;
+
+            const componentPath = resolve(`${outDir}/${componentShortName}`);
+            await outputFile(
+                resolve(`${componentPath}/index.ts`),
+                prettier.format(componentSrc, {
+                    parser: 'babel',
+                    printWidth: 80,
+                    tabWidth: 4,
+                    semi: true,
+                    singleQuote: true,
+                    trailingComma: 'es5',
+                    bracketSpacing: true,
+                    arrowParens: 'always',
+                    htmlWhitespaceSensitivity: 'ignore',
+                })
+            );
+            await outputFile(
+                resolve(`${componentPath}/package.json`),
+                packageJson
+            );
+            await outputFile(
+                resolve(`${componentPath}/tsconfig.json`),
+                tsconfigJson
+            );
```

Comment:
Here we create all of the files for the React packages _every_ time, right? Does that point to there being a benefit in simply `.gitignore` listing these build artifacts, so that we don't have to track this output in the repo over future PRs?

Reply:
Sounds good to me. I removed all the generated artifacts from this PR and added `react` folder to the gitignore file.